packages/playwright-core/src/server/registry/index.ts

```diff
@@ -1315,6 +1315,8 @@ export async function installBrowsersForNpmInstall(browsers: string[]) {
     return false;
   }
   const executables: Executable[] = [];
+  if (process.platform === 'win32')
+    executables.push(registry.findExecutable('winldd')!);
```

Comment:
why do we have this for winldd but not for ffmpeg?

Reply:
This function is used for the browser-* packages which will install the browsers during the NPM install. We specify `ffmpeg` there every time on the caller side - we don't want to do that with `winldd` since its based on the `process.platform`.